x-pack/packages/security-solution/upselling/service/index.ts

```diff
@@ -10,4 +10,5 @@ export type {
   SectionUpsellings,
   UpsellingSectionId,
   UpsellingMessageId,
+  MessageUpsellings,
```

Comment:
Is this export needed? It does not seem to be used

Reply:
It is not used in this PR but it is a missing import from a [previous PR](https://github.com/elastic/kibana/pull/163406). I was seeing a type error in my PR cause of it.